tracks/clojure/exercises/reverse-string/mentoring.md

```diff
@@ -9,6 +9,13 @@ This exercise is about re-implementing a common library function.
 ```clojure
 (ns reverse-string)
 
+(defn reverse-string [s]
+  (apply str (reverse s)))
+```
+
```

Comment:
What you wrote in the PR description might be helpful to add to the mentoring notes, as to why one might prefer this over another solution.  And what comments might be made while mentoring this exercise.